### ISSUE TITLE
[5.1] [a11y] Dashboard module buttons missing focus styling

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#43066]
### Steps to reproduce the issue

When tabbing through the elements in the Joomla dashboard, there is no focus styling on buttons (with a cog icon), meaning that the user won't know they're focused on said element.

![image](https://github.com/joomla/joomla-cms/assets/2019801/07721558-b6f0-4a17-a8f1-a07f3696b24c)

**Comment by brianteeman** [comment]
Well spotted - however this is not a dark mode issue as the same problem exists in light mode and has done for some time.

**Comment by brianteeman** [comment]
please test #43135
